application/lib/console.dart

```diff
@@ -111,7 +119,45 @@ class _ConsoleState extends State<Console> {
               _buildChip('take all'),
               _buildChip('drop all'),
               _buildChip('debug status'),
-              _buildChip('debug locations'),
+              OutlinedButton(
+                child: const Text('Teleport'),
```

Comment:
convention is to have the label end with "..." if it's going to do something like bring up a dialog